README.md

```diff
@@ -1,65 +1,128 @@
 # Data Exporter (post-release)
 
-This program will produce data exports for submission to NCBI, UCSC, and Europe PMC.  Files provided to these resources are:
-* **NCBI:** Gene (NCBI gene identifier to Reactome top level pathways) and Protein (all UniProt entries in Reactome associated with any NCBI Gene identifier).
-* **UCSC:** Entity (UniProt entries in Reactome) and Events (UniProt entries in Reactome in relation to Reactome pathways and reactions).
-* **Europe PMC:** Profile and Link XML (describing Reactome as a provider and Reactome Pathway to literature references, respectively).
-
-The file outputs will be as follows in the configured output directory (see configuration section below) and where XX is the Reactome Version:
-
-* gene_reactomeXX-Y.xml (where Y is the file number as this file is split between multiple files so the file upload size is acceptable for NCBI)
+This program will produce data exports for submission to NCBI, UCSC, and Europe PMC.  Files provided to these resources
+are:
+* **NCBI:** Gene (NCBI gene identifier to Reactome top level pathways) and Protein (all UniProt entries in Reactome
+ associated with any NCBI Gene identifier).
+* **UCSC:** Entity (UniProt entries in Reactome) and Events (UniProt entries in Reactome in relation to Reactome 
+pathways and reactions).
+* **Europe PMC:** Profile and Link XML (describing Reactome as a provider and Reactome Pathway to literature 
+references, respectively).
```

Comment:
Do you think it would be a good idea to add a linkout to each of the mentioned resources?

Reply:
Yeah, I didn't think of that, but you're right.  I'll try to find and add the appropriate links.

Reply:
Done: https://github.com/reactome/release-data-exporter/pull/4/commits/4d9f7eb2148afd182d742613874ec1873a815284